README.md

```diff
@@ -38,8 +38,8 @@ $ sbt
 > exit
 $ cd target/sbt-test/default-*/scripted
 $ sbt
-$ jetty:start
-$ browse # starts browser for you, or manually open http://localhost:8080 to verify
+> jetty:start
+-> executes browser, and opens `http://localhost:8080` to verify
```

Comment:
I'm sorry.
My operation was wrong ...
However, since the path names are different, we will fix that.